tests/test_webhook.py

```diff
@@ -177,6 +182,17 @@ def test_extract_priority_label(self):
         priority_label_none = helpers.extract_priority_label(self.issue_body)
         self.assertEqual(priority_label_none, None)
 
+    def test_get_issue_labels(self):
+        """Extract list of labels from an issue body."""
+        labels_tests = [
+            (self.issue_body, ['browser-firefox', 'type-media', 'type-stylo']),
+            (self.issue_body2, ['type-foobar']),
+            (self.issue_body3, ['browser-firefox-mobile-tablet'])
+        ]
+        for issue_body, expected in labels_tests:
+            actual = helpers.get_issue_labels(issue_body)
+            self.assertEqual(sorted(expected), sorted(actual))
```

Comment:
FWIW, according to Python docs, speaking of `assertListEqual` and `assertTupleEqual`:

`These methods are used by default when comparing lists or tuples with assertEqual().`